espanol/granhermano/granhermano.md

```diff
@@ -0,0 +1,5 @@
+La guerra es la paz. La libertad es la esclavitud. La ignorancia es la fuerza 
```

Comment:
I have made the fixes in my local master, should I push the master now?

Christopher